backfill_corrections/delphiBackfillCorrection/man-roxygen/testing_window-template.R

```diff
@@ -0,0 +1,2 @@
+#' @param testing_window the testing window used for saving the runtime. Could
+#'     set it to be 1 if time allows
```

Comment:
I would suggest to train it every 2 weeks or every 4 weeks

Reply:
Since we run it daily, we can just set it to 1. If for some reasons we don't have the prediction result for several days and don't traverse the training date, then we can produce the prediction for those days at once by change the value of testing window

Reply:
To what ever number of days of prediction we missed.